.env.rc1

```diff
@@ -289,7 +289,7 @@ VOTING_BOT_CRON_SCHEDULE="1 * * * *"
 # --- Komenci ---
 
 KOMENCI_DOCKER_IMAGE_REPOSITORY="celotestnet.azurecr.io/komenci/komenci"
-KOMENCI_DOCKER_IMAGE_TAG="d78d4c25fcd04871a81c67e6107318c9dbd7ec03"
+KOMENCI_DOCKER_IMAGE_TAG="ef1cf7bc7c7ade7de8188a59f73c275771a14771"
```

Comment:
Are you sure about this commit? It looks like a merge commit that is a part of a bigger change that has been already merged.

Reply:
Done.